docs/ISSUE_TRIAGE.md

```diff
@@ -0,0 +1,199 @@
+4. Consider requesting [help from the community](#5-requesting-help-from-the-community).
+
+### Low priority
+
+1. If the issue is deemed possibly useful but is a low priority, then label the issue `priority/low`.
+2. The functional impact of an issue determines how high of a priority the issue is.
+3. Consider requesting [help from the community](#5-requesting-help-from-the-community).
+
+## 5. Requesting help from the community
+
+Due to constraints, maintainers aren't always able to resolve issues in a timely manner, especially those of lower priority. For such issues, consider requesting help from the community. Use your best judgement. In general, requesting help from the community means that a contribution has a good chance of getting accepted and merged.
+
+In many cases, the issue author or the community is most suitable to contribute changes since they're experts in their domain. Also, it's very common for someone to seek help from the community on a problem.
+
+1. Kindly and politely add a comment which will notify the users subscribed to the issue of any updates.
+   - Explain that the issue would be nice to get resolved, but it isn't prioritized to be worked on by maintainers for the foreseeable future.
+   - If possible or applicable, try to help contributors get started by adding pointers to relevant source code or files as well as adding any helpful input on how to resolve the issue.
+2. Label the issue with `help wanted`.
+3. If applicable, label the issue with `beginner friendly` to denote that the issue is suitable for a beginner to work on.
+
+## Investigation of issues
+
+When an issue has all basic information provided, but the reported problem cannot be reproduced at a first glance, the issue is labeled `triage/needs-information`. Depending on the perceived severity and/or number of upvotes, the investigation will either be delegated to another maintainer for further investigation or put on hold until someone else (maintainer or contributor) picks it up and eventually starts investigating it.
+
+Even if you don't have the time or the knowledge to investigate an issue we highly recommend that you upvote the issue if you happen to have the same problem. If you have further details that may help with investigating the issue please provide as much information as possible.
+
+## External pull requests
+
+Part of issue triage should also be triaging of external PRs. The main goal should be to make sure PRs from external contributors have an owner/reviewer and are not forgotten.
+
+1. Check new external PRs which do not have a reviewer.
+1. Check if there is a link to an existing issue.
+1. If not and you know which issue it is solving, add the link yourself, otherwise ask the author to link the issue or create one.
+1. Assign a reviewer based on who was handling the linked issue or what code or feature does the PR touches (look at who was the last to make changes there if all else fails).
+
+## GitHub issue management workflow
```

Comment:
Was there intended to be more content here? The "ask a question" and "new feature" types aren't shown, and no actual workflow steps are described.

Reply:
@kerry-meyer  :  As of now there is no additional content. If you have any suggestion for adding more content related to "Ask a Question" and/or "New Feature" types, kindly provide the content. 
Alternatively, you can approve this with the current content if that is acceptable.

Reply:
Please at least provide the proposed additional steps of the workflow expected for handling of a "Bug" type issue (leading up to and including closing of the issue) and/or add a comment indicating that this is a placeholder for future content and stating that maintainers will provide additional details in the future. 

Alternatively, this section can be deleted since it is mostly already covered by the previous content of the document. Since the full workflow is beyond the scope of "triage", it could, and probably should be included in one of the other documents ("maintainer's guide" or "contributor's guide") if it's not already there and referenced here instead of providing a partial section.

Reply:
I don't think we need this. Let's just delete it. 
If it's strictly required for OneAnsible, we can use the same content that other components have been using. Otherwise let's just get rid of it; it's extraneous detail.

Reply:
@jeff-yin This requirement is coming from OneAnsible. I am not clear on what you mean by content from "other components". Which other components are you referring to?

Reply:
@jeff-yin : This document is the guide for "core" maintainer/developer/lead who will do the first round of triaging. Do you mean to remove this document completely? Or, do you mean to remove this particular point?

Reply:
"other components" -- just meaning Storage and Servers.

I just mean to remove this particular point, not the entire section. I wrote this in reply to this thread about the "GitHub issue management workflow" section, so I'm not sure why it was interpreted that I meant to remove the document completely.

Reply:
Server/storage team is planning to create a workflow diagram and add to this document later. Until then, the section "##GitHub issue management workflow" is not required and hence removed. This document will be enhanced in future version when the diagram is available.

Reply:
Because real content has been provided, I am marking this issue "Resolved".